stock_request/models/stock_request.py

```diff
@@ -34,7 +29,7 @@ def _get_expected_date():
 
     name = fields.Char(states={"draft": [("readonly", False)]})
     state = fields.Selection(
-        selection=_get_request_states,
+        selection="_get_states",
```

Comment:
@victoralmau I think you should use a lambda method instead

Reply:
@victoralmau Please follow @chienandalu comments and change this to lambda? Thanks